modules/meta.analysis/R/run.meta.analysis.R

```diff
@@ -165,12 +176,12 @@ run.meta.analysis.pft <- function(pft, iterations, random = TRUE, threshold = 1.
 ##' and post.distns.Rdata, respectively
 ##' @export
 ##' @author Shawn Serbin, David LeBauer
-run.meta.analysis <- function(pfts, iterations, random = TRUE, threshold = 1.2, dbfiles, database) {
+run.meta.analysis <- function(pfts, iterations, random = TRUE, threshold = 1.2, dbfiles, database, use_ghs = FALSE) {
   # process all pfts
   dbcon <- db.open(database)
   on.exit(db.close(dbcon))
 
-  result <- lapply(pfts, run.meta.analysis.pft, iterations, random, threshold, dbfiles, dbcon)
+  result <- lapply(pfts, run.meta.analysis.pft, iterations, random, threshold, dbfiles, dbcon, use_ghs)
```

Comment:
I'd be inclined to make all of the fixed arguments (`iterations` on) explicitly named, so that we don't introduce subtle bugs if we ever change the argument order in `run.meta.analysis.pft`. But that may just be me being paranoid.

Reply:
I personally prefer that too